frontend/src/bundles/common/components/base-layout/styles.module.css

```diff
@@ -3,7 +3,7 @@
         "header header"
         "aside main";
 
-    @apply grid grid-cols-[256px,auto] grid-rows-[88px,auto] overflow-hidden transition-all duration-[0.5s] ease-[ease-in-out] lg:grid-cols-[288px,auto];
+    @apply grid grid-cols-[256px,auto] grid-rows-[88px,auto] overflow-hidden transition-all duration-[0.5s] ease-[ease-in-out] sm:grid-cols-[0,auto] lg:grid-cols-[288px,auto];
```

Comment:
Could you please use rem not px

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/78347072/615d82c2-f7fe-48de-ab69-35be167456f1)
remove these rules